harness/tests/experiment/pytorch/test_reducer.py

```diff
@@ -12,3 +23,96 @@ def test_reducer() -> None:
 
     batches_per_process = [1, 2, 5, 4, 5, 6]
     assert np.around(_reduce_metrics(Reducer.AVG, metrics, batches_per_process), decimals=2) == 6.43
+
+
+DummyTrial = namedtuple("DummyTrial", "distributed_context reducer_context wrapped_reducer")
```

Comment:
Since Trial is of particular meaning, perhaps rename this to something without "Trial"?  Don't waste too much time on it though.